test/ci/create_aws_vm.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Prepare aws_instance tag
+  set_fact:
+    aws_instance_tags: |
+      {
+        "kubernetes.io/cluster/{{ aws_cluster_id }}": "true",
+      }
+
+- name: Create a VM
+  ec2:
+    region: "{{ aws_region }}"
+    key_name: "{{ aws_key }}"
+    instance_type: "{{ outer_item.aws_flavor }}"
+    image: "{{ outer_item.aws_image | default(aws_image) }}"
+    wait: yes
+    group: "{{ outer_item.aws_security_group }}"
+    count: 1
+    vpc_subnet_id: "{{ aws_subnet }}"
+    assign_public_ip: yes
+    instance_tags: "{{ aws_instance_tags |
+      combine({'Name': outer_item.name_template + '-' + item }) }}"
+  register: ec2
+  with_sequence: start=1 end={{ outer_item.amount }}
+
+- name: Attach the volume for gluster
+  ec2_vol:
+    instance: "{{ item.instances.0.id }}"
+    name: "{{ item.instances.0.tags.Name }}-volume"
+    device_name: /dev/sdc
+    volume_size: 50
+    delete_on_termination: yes
+    region: "{{ aws_region }}"
+  when: "'glusterfs' in outer_item.groups"
+  with_items: "{{ ec2.results }}"
+
+- name: Attach the volume for Atomic
```

Comment:
Why do something different for atomic?

Reply:
Default AMI for Atomic is 5GB only, so docker pulls would break. Also, Atomic needs `/dev/sdb`, while gluster nodes should have `/dev/sdc` so that default docker volume on test AMIs won't be replaced.

Perhaps `volumes` should be a config param same as AMI for each node group?

Reply:
Yeah, we can use the volumes arg to ec2 for this as well.  Just supply a volume list for each type of install we're doing.

Reply:
Ditto